scripts/helm.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# All HELM related common logic should be kept here
+
+function _set_variables() {
+
+  ## internal func: Sets variables needed for running helm pacakge
+
+  echo "Intialising variables needed for HELM Publish"
+
+  APP_NAME=$(echo $TRAVIS_REPO_SLUG | cut -d"/" -f2)
+
+  if [[ -z "$TRAVIS_TAG" ]]; then
+      VERSION=$(date +%y.%m.%d)-${TRAVIS_COMMIT}
+      APP_VERSION=${TRAVIS_COMMIT}
+  else
+    VERSION=${TRAVIS_TAG}
+    APP_VERSION=${TRAVIS_TAG}
+  fi
+
+}
+
+function _validate_variables() {
+
+  ## internal func: Validates all required variables exist
+
+  echo "Validating input variables needed for HELM Publish"
+
+  if [ -z "$ARTIFACTORY_URL" ]; then
```

Comment:
There should be a new variable for the helm URL. Since docker and non-docker repositories in nexus have different URLs, docker.quid.com and nexus.quid.com, we need to have both of them set in the build configuration.

Reply:
@rsyabuta Not sure I understood this, isn't ARTIFACTORY_URL=nexus.quid.com. Or you're suggesting that we should have one more variable like HELM_URL=nexus.quid.com dedicated to HELM. 

May be we can also do like:
ARTIFACTORY_SUB_DOMAIN = nexus.quid.com
ARTIFACTORY_HELM_OATH = /repository/quid-helm/quid/

Reply:
ah! updated.